script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -41,7 +40,7 @@ A flying ice spear is nothing more than a flung droplet of water before this wal
 -- Page 661, Offset 10428.
 -- 高さ50メートル、7000トンに相当する質量が、ヴローヴを飲み込まんと押し寄せる。
 // 高さ50メートル、7000トンに相当する質量が、ヴローヴを飲み込まんと押し寄せる。
-The mass, 50 meters tall and weighing 7000 tons, bears down on Vlov as if to swallow him whole.
+The mass, fifty meters tall and weighing 7000 tons, bears down on Vlov as if to swallow him whole.
```

Comment:
Feels weird to switch mid-sentence, keep both numbers

Reply:
I actually feel like writing out "seven thousand tons" gives it more weight but I don't feel that strongly